Source/Core/VideoBackends/OGL/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ PUBLIC
   videocommon
 
 PRIVATE
-  ${X11_LIBRARIES}
+  PkgConfig::X11
```

Comment:
Looks like the reason for the macOS failure is because `${X11_LIBRARIES}` used to expand to nothing there. Now `PkgConfig::X11` tries to add a dependency on a target that doesn't exist.

It's probably the same on Android.